clang/test/dpct/wmma.cu

```diff
@@ -0,0 +1,154 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0
+// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0
+// RUN: dpct --format-range=none --use-experimental-features=oneapi-joint-matrix -out-root %T/wmma %s --cuda-include-path="%cuda-path/include" -- -std=c++14 -x cuda --cuda-host-only
```

Comment:
Please also add a test case that option:  --use-experimental-features=oneapi-joint-matrix   is not used.